cpimauth_settings.php

```diff
@@ -0,0 +1,276 @@
+<?php
+class cpimauth_settings_page
```

Comment:
What does `settings_page.php` do, then?

---
In reply to: [71237235](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71237235) [](ancestors = 71237235,71235409)